aws_doc_sdk_examples_tools/metadata_test.py

```diff
@@ -154,8 +157,22 @@ def test_parse():
             "sqs": set(),
         },
         doc_filenames=DocFilenames(
-            sdk_pages=["sns_DeleteTopic_cpp_1_topic"],
-            service_page=None,
+            service_pages={
+                "sns": "https://docs.aws.amazon.com/code-library/latest/ug/sns_example_sns_DeleteTopic_section.html",
```

Comment:
These tests are really hard to read, can you extract out the links into a function that makes it obvious what they're linked to?

`delete_topic_file_name('sns'),`
`cpp_code_example('sns'),`

Reply:
Does this make it more clear?
```
        doc_filenames=DocFilenames(
            service_pages={
                "sqs": make_doc_link(stub="sqs_example_sns_TestExample_section"),
                "sns": make_doc_link(stub="sns_example_sns_TestExample_section"),
            },
            sdk_pages={
                "java": {
                    2: SDKVersion(
                        actions_scenarios={
                            "sqs": make_doc_link(stub="java_2_sqs_code_examples", anchor="scenarios"),
                            "sns": make_doc_link(stub="java_2_sns_code_examples", anchor="scenarios"),
                        }
                    )
                },
```

Reply:
Updated.